src/App2.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from "react"
```

Comment:
@myteax - Did you mean to have this second App.js file?

Reply:
App2.js is the class component you did.  It can be deleted if you do not need it again